sew.lic

```diff
@@ -107,6 +107,7 @@ class Sew
 
   def study_tailoring(chapter, recipe)
     get_item('tailoring book')
+    echo('*** You will need to upgrade to a journeyman or master book before 176 ranks! ***') if DRSkills.getrank('Outfitting') == 175
     bput("turn my book to page #{find_recipe(chapter, recipe)}", 'You turn your', 'The book is already')
```

Comment:
💡 I like the exit-early idea. Would that be easier if `find_recipe` returned a bool?